client/package.json

```diff
@@ -19,7 +19,7 @@
     "http-status-codes": "^1.4.0",
     "moment": "^2.27.0",
     "remove-accents": "^0.4.2",
-    "sbc-common-components": "2.4.5",
+    "sbc-common-components": "2.4.6",
```

Comment:
Why does line 3 still show 0.1.6 ? It should show 0.2.0. And then I was going to say, please bump it up to 0.2.1 :)

Reply:
Copy that! Will do

Reply:
As there is no 0.2.0 yet, just change it to that.